academic_observatory_workflows/workflows/openalex_telescope.py

```diff
@@ -300,13 +371,66 @@ def __init__(
 
         self.add_setup_task(self.check_dependencies)
         self.add_task(self.write_transfer_manifest)
-        # self.add_task(self.upload_transfer_manifest) #TODO uncomment when using transfer manifest
+        # self.add_task(self.upload_transfer_manifest)
         self.add_task(self.transfer)
         self.add_task(self.download_transferred)
         self.add_task(self.transform)
         self.add_task(self.upload_transformed)
-        self.add_task(self.bq_load_partition)
-        self.add_task_chain([self.bq_delete_old, self.bq_append_new, self.cleanup, self.add_new_dataset_releases])
+        self.add_task(self.bq_append_new)
+        self.add_task(self.bq_delete_old)
+        self.add_task(self.bq_create_snapshot)
+        self.add_task(self.add_new_dataset_releases)
+
+    def get_release_info(self, **kwargs) -> Tuple[pendulum.DateTime, pendulum.DateTime, bool]:
+        """Return release information with the start and end date.
+        The start date is customised compared to the parent class to use the 'created' date from the latest dataset
+        release. This is because there are multiple dataset releases for one OpenAlexRelease. Each dataset release
+        maps to one of the OpenAlex entities and has a different end date, the created date should be the same for
+        all of them.
+
+        :param kwargs: The context passed from the PythonOperator.
+        :return: None.
+        """
+        # Check if first release or not
+        first_release = is_first_release(self.workflow_id)
+        if first_release:
+            # When first release, set start date to the start of the DAG
+            # Can we get away with using data_interval_start instead?
+            start_date = pendulum.instance(kwargs["dag"].default_args["start_date"]).start_of("day")
+        else:
+            datasets = get_datasets(workflow_id=self.workflow_id)
+            releases = get_dataset_releases(dataset_id=datasets[0].id)
+            latest_release = get_latest_dataset_release(releases=releases)
+            start_date = pendulum.instance(latest_release.created).start_of("day")  # custom start date
+
+        end_date = pendulum.today(tz="UTC").start_of("day")
```

Comment:
Can the end date not be the same as get_release_info from StreamTelescope? end_date = pendulum.instance(kwargs["data_interval_end"])